### DISCUSSION TITLE
Filter (search box) in extension list

### DISCUSSION THREAD

**Discussion by johanpeters** [akeeba/panopticon#248] [Ideas]
currently al extentions found are in a long list. ofcourse i can use the search option of my browser but would be nice to have filter options / search option on the extention list.

i am talking about the system administration menu and then the tab automatics task.
the filter on name (select of based on text entry), author of extention (filter of text entry), filter on prefference (only select).

same request on the page of a specific website.

**Comment by nikosdion** [comment]
There can be only one filter field which searches both name and author. This has to happen client-side, with JavaScript. Overcomplicating it is not a good idea.

Filtering by preference is a bad idea, since you're on the page where you can change that preference. This leads to a surprising interface, in the negative sense of “it catches the user by surprise”. You see an extension, click on the drop-down select, choose a different option… and the entire row disappears, without giving you any kind of confirmation of your choices. If it does not do that, then the filter state and the display state are at odds. Half of the people will be upset in the former case, half of the people will be upset in the latter.

**Comment by johanpeters** [comment]
okay.. clear rgarding filter.. only a search field would be nice.. 
defently not a dealbreaker.. just makes is easier to find specific things easier..

**Comment by nikosdion** [comment]
OK! Will do. Can't promise an exact date as I need to make a release this week to address installation issues.

**Comment by nikosdion** [comment]
Now tracking this in #247 and converting this issue to a Discussion.
